docs/docs/guides/web3_migration_guide/web3_treeshaking_guide.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 11
+sidebar_label: Web3.tree.shaking
```

Comment:
lets put this under new `web3 tree shaking support guide` instead of under migration guide.

Reply:
moved